.github/CODEOWNERS

```diff
@@ -8,4 +8,4 @@
 *               @googleapis/yoshi-python
```

Comment:
Did you mean to add `@googleapis/cloud-storage-dpe` here as well?
```suggestion
*               @googleapis/yoshi-python @googleapis/cloud-storage-dpe
```

Reply:
Ah yes, thank you